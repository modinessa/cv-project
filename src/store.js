import { configureStore } from "@reduxjs/toolkit";
import { navigationPanelReducer } from "./components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationPanelReducer,
	}
});
