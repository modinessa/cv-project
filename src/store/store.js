import { configureStore } from "@reduxjs/toolkit";
import navigationReducer from "../components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationReducer,
	}
});