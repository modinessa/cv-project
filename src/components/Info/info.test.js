import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

import { store } from '../../store/store';
import { Info } from './index';

const TEST_TITLE = 'Title';
const TEST_TEXT = 'Content bla-bla';

describe('Info', () => {
	it('renders component', () => {
		expect(renderer.create(
			<Provider store={store}>
				<Info
					title={TEST_TITLE}
					text={TEST_TEXT}
				/>
			</Provider>
		)).toMatchSnapshot();
	})
})