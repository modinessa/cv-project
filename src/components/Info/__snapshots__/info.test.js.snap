// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info renders component 1`] = `
<div
  className="info"
>
  <h3>
    Title
  </h3>
  <p>
    Content bla-bla
  </p>
</div>
`;
